components/brave_wallet_ui/common/slices/wallet.slice.ts

```diff
@@ -606,5 +610,18 @@ export const createWalletReducer = (initialState: WalletState) => {
 
 export const walletSlice = createWalletSlice()
 export const walletReducer = walletSlice.reducer
+export const persistedWalletReducer = persistVersionedReducer(
+  walletReducer,
+  {
+    key: 'wallet',
+    version: PERSISTED_STATE_VERSION,
+    blacklist: [
```

Comment:
Same question here, where can I see all the potential ones can be added here? Thinking it would be good to go allow list here at the beginning until we have a better understanding of which data is being persisted

Reply:
List is available here: https://github.com/brave/brave-core/blob/feat-persist-wallet-redux-store/components/brave_wallet_ui/constants/types.ts#L229

Reply:
updated to a whitelist